### ISSUE TITLE
Add DragDropContainer data to onDrop Callback 

### ISSUE THREAD

**Issue by herbertpimentel** [peterh32/react-drag-drop-container#25]
onDrop: Callback must also receive in its object argument the data of DragDropContainer 

this way we can easily in a single event know wich DragDropContainer.Data was dragged to wich DropTarget.Data

I can go around it with something like:

```javascript
  dropOk = (dragData, dropData) => {
    console.log("draggable data", dragData, "targed data", dropData);
   
    // rest.changeElementToList(dragData.id, dropData.id);
  };

```

```jsx

 <DragDropContainer
          dragData={{ id: 1 }}
          onDrop={e => this.dropOk({id: 1}, e.dropData)}
        >
          <span>dragbable 1</span>
  </DragDropContainer>


<DropTarget dropData={{ id: "list_todo" }}>
          <div>Todo List</div>
  </DropTarget>

<DropTarget dropData={{ id: "list_done" }}>
          <div>Done List</div>
  </DropTarget>


````

**Comment by peterh32** [comment]
I'm slightly confused -- when you successfully drop a `DragDropContainer` onto a `DropTarget`, the `DragDropContainer` gets back an `onDrop` event containing `dropElem`, which is the `dropTarget` element it landed on. 

Why would `DragDropContainer` need to get its own self back in the `onDrop` event? It knows who it is, doesn't it?

**Comment by herbertpimentel** [comment]
Yep, but `dropElem` is an reference to the DOM object. I was expecting an easy way to get the `dropData` of this element.

**Comment by peterh32** [comment]
OK! Fixed in release 6.0.0. It now passes
- dropData
- dropElem
- dragData
- target
...plus other standard react/js event data
